.gitignore

```diff
@@ -24,6 +24,7 @@ bld/
 [Oo]bj/
 [Ll]og/
 [Tt]emp/
+**/packages.lock.json
```

Comment:
did you try using the central lockfile? or does it just feel like lockfiles still change too often to be pleasant to use here?

Reply:
lock files are always generated for central package management